### PR TITLE
Upgrade configurations to python 3.7

### PR DESCRIPTION
**Pull request by madhuBayy**
@somexpert  @ldko This project doesn't seem to require changes in source code to upgrade it to python3. I have updated the dependencies and configuration settings to python3 and the tests worked fine. Please review this PR.

### CODE REVIEW

File: .travis.yml
Comment (somexpert) [inline .travis.yml]: Gotta change all these env names too :)

**Review by somexpert — APPROVED** [review]
Looks good :thumbsup:

**Comment by somexpert** [comment]
oh, also, if there are no backwards-incompatible changes, then couldn't we just merge this into master?

**Comment by madhuBayy** [comment]
Well, it can be merged to master. But I'm not sure if the applications that use `django-edtf` are on python3, 'cos `django-edtf` has a dependency on `edtf-validate`.

**Comment by somexpert** [comment]
oh, right, forgot that we're getting the py3 branch of that dependency. nvm then

File: requirements.txt
Comment (ldko) [inline requirements.txt]: This should stay master, I am waiting for an approval on edtf-validate, and then it will be merged to master as it still works in py2 as well.

File: setup.py
Comment (ldko) [inline setup.py]: should leave this with master and wait to merge until the edtf-validate pr that is open is merged.

File: setup.py
Comment (ldko) [inline setup.py:24]: If this still supports Python 2.7 and other versions of Python 3, then those could be left listed here as well.

File: README.md
Comment (ldko) [inline README.md:22]: Does this require specifically Python 3.7? We can list more than one version of Python if this supports more than one version.

File: .travis.yml
Comment (ldko) [inline .travis.yml:13]: If we are planning to merge this to master because it still supports Python 2, I suggest we still test against python 2.7 as well. What do you think @madhulika95b  @somexpert

File: tox.ini
Comment (ldko) [inline tox.ini:5]: Again, should we also test Python 2.7?

File: .travis.yml
Comment (somexpert) [reply .travis.yml:13]: normally I'd say that's fine, but since we're mere months away from Python 2 being completely deprecated, I'm not sure it's worth it. I'm not set on that mindset, but that's where I'm leaning personally

File: setup.py
Comment (madhuBayy) [reply setup.py]: ok!

File: .travis.yml
Comment (ldko) [reply .travis.yml:13]: I would agree with that if we weren't still running this in Python 2 for now.

Comment (ldko) [reply .travis.yml:13]: Maybe not add back to travis then, but maybe tox?

Comment (madhuBayy) [reply .travis.yml:13]: Are you suggesting to test it in both python2.7 and python 3.7 ? And also in just travis or in tox as well? 
Edit: Ah, Just read the comment below.

Comment (somexpert) [reply .travis.yml:13]: There's certainly no harm in doing that, or even in adding them back to Travis.

**Comment by madhuBayy** [comment]
Also, Is it necessary to run the tests in django 1.9 & django 1.10 anymore?

**Comment by ldko** [comment]
If we already have this deployed in Django 1.11 then I think we can drop tests for Django 1.9 and 1.10, since they aren't supported anymore.

**Comment by madhuBayy** [comment]
Shall I merge it directly to master as @somexpert suggested when you all think it's ready to get merged?

**Comment by somexpert** [comment]
I _think_ it makes sense, since django-edtf and its edtf-validate dependency are both compatible with Python 2.7 and 3+ on their master branches. But I may be overlooking something. What do you think, @ldko?

**Comment by ldko** [comment]
I think we should merge this into `master`.

**Review by ldko — APPROVED** [review]
Works for me. The PR should be changed to merge into `master`. :+1:

**Pull request merged**
